driver-core/src/main/com/mongodb/connection/DefaultServerConnection.java

```diff
@@ -117,17 +120,14 @@ public BulkWriteResult insertCommand(final MongoNamespace namespace, final boole
     @Override
     public BulkWriteResult insertCommand(final MongoNamespace namespace, final boolean ordered, final WriteConcern writeConcern,
```

Comment:
I'm torn about these methods.  We have three options:

* Deprecate them but leave them as you have them here
* Throw UnsupportedOperationException, essentially breaking compatibility and breaking any application that uses Connection directly.  I think the number of applications that use Connection directly is zero.
* Just remove the methods, breaking binary compatibility entirely.  See above.

I'm inclined more towards the latter.  It's not really different in practice from option 2, just a different exception.